README.md

```diff
@@ -54,73 +54,73 @@ The following plugins are considered alpha and still under heavy development. Th
 
 ## New Frontend System
 
-The Demo site currently uses the currently default Legacy Frontend System as of 2025-05-19 as the New Frontend System is still currently in Alpha. For those looking for a working example of a migrated "app" we have added an `app-migrated` to showcase this. You can see the code for this in `/packages/app-migrated` and you can run this version locally by running: `yarn start:app-migrated`.
+The Demo site uses the default Legacy Frontend System as of 2025-05-19, as the New Frontend System is still in Alpha. For those looking for a working example of a migrated "app", we have added an `app-migrated` to showcase this. You can see the code for this in `/packages/app-migrated` and you can run this version locally by running: `yarn start:app-migrated`.
 
 The `app-migrated` example is in a state where the ["Switching out `createApp`"](https://backstage.io/docs/frontend-system/building-apps/migrating/#switching-out-createapp) and the ["Migrating `createApp` Options"](https://backstage.io/docs/frontend-system/building-apps/migrating/#migrating-createapp-options) steps of the [Migrating Apps](https://backstage.io/docs/frontend-system/building-apps/migrating/) guide have been completed.
 
 ## Code Customization
 
-We have made some code customization as well. The following sections go into detail about them.
+We have made some code customizations as well. The following sections go into detail about them.
 
 ### Custom Theme
 
-We have created a custom theme called Aperture to help showcase what some of the possibilities are that a custom theme can allow and to have a working example.
+We have created a custom theme called `APERTURE` to help showcase what some of the possibilities are that a custom theme can allow and to have a working example.
```

Comment:
We do actually call it Aperture: 

https://github.com/backstage/demo/blob/d1bca7fa8cdb88e5fb24f706dc20937c3902b228/packages/app/src/App.tsx#L116

It just get formatted in the UI to be upper case.